docs/adr/0002-custom-dev-container-image.md

```diff
@@ -0,0 +1,34 @@
+# 2. Provide a custom dev container image
+
+Date: 2024-06-04
+
+## Status
+
+Accepted
+
+## Context
+
+We are using [dev containers](https://code.visualstudio.com/docs/devcontainers/create-dev-container) to provide a configuration for Codespaces that gives researchers a useful development environment to use while working on their OpenSAFELY projects. We want to improve that environment to make it more appealing and useful to researchers.
+
+## Decision
+
+We will have a single dev container environment with a custom Docker image, defined within the [research-template](https://github.com/opensafely/research-template/blob/main/.devcontainer/devcontainer.json) repo, available for researchers to use within Codespaces.
```

Comment:
I don't think this decision relates to the title of the ADR, which is about the decision to use a custom image. The decisions to use one, rather than many, dev container configurations, and to locate the dev container configuration within opensafely/research-template are orthogonal.